### ISSUE TITLE
Invalid property "test" on appJsonObject

### ISSUE THREAD

**Issue by hemisphera** [Microsoft/navcontainerhelper#189]
https://github.com/Microsoft/navcontainerhelper/blob/893577764550b2f4d8ded89959ab911dd3e173cf/AppHandling/Compile-AppInNavContainer.ps1#L77

The cmdlet `Compile-AppInNavContainer` references an unknown property "test" on the object appJsonObject. This does not seem to be part of the specificiation. Might be "just" a debug leftover? This errors out on me with the following message during build:

System.Management.Automation.PropertyNotFoundException: The property 'test' cannot be found on this object. Verify that the property exists.

**Comment by freddydk** [comment]
I guess you are using a clone from the github repo as this stems from a PR which isn't yet published on the gallery.
I will fix this to test for the test property before using it.
Thanks

**Comment by hemisphera** [comment]
Exactly. I am using a clone of my fork on our build server which enables me to debug and have the latest changes at hand without having to wait for a push to the gallery.

Out of curiosity: I have not found any documentation regarding `test` on the official docs. What is this property for?

**Comment by freddydk** [comment]
I think the 4 lines should be replaced by this
```
    if (([bool]($appJsonObject.PSobject.Properties.name -match "test")) -and $appJsonObject.test)
    {
        $dependencies +=  @{"publisher" = "Microsoft"; "name" = "Test"; "version" = $appJsonObject.test }
    }

```

**Comment by freddydk** [comment]
I will get back to you on your question
